### PR TITLE
15281 Remove full restoration for COOPs from allowable actions

### PR DESCRIPTION
**Pull request by argush3**
*Issue #:* /bcgov/entity#15281

*Description of changes:*

* Remove full restoration filing from allowable actions for historical COOPs.  This was in place from before but with FE now using allowable actions to drive what is displayed, this will be removed.
* Added/updated tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

**Comment by argush3** [comment]
Note: Kial will be fixing the linting error and broken test.

File: legal-api/tests/unit/services/test_authorization.py [hunk 1]
```diff
@@ -1513,8 +1509,7 @@ def mock_auth(one, two):  # pylint: disable=unused-argument; mocks of library me
          expected_lookup([FilingKey.COURT_ORDER,
                           FilingKey.PUT_BACK_ON,
                           FilingKey.REGISTRARS_NOTATION,
-                          FilingKey.REGISTRARS_ORDER,
-                          FilingKey.RESTRN_FULL_CP])),
```
Comment (severinbeauvais) [inline legal-api/tests/unit/services/test_authorization.py:1517]: Can you delete this key now?

**Review by severinbeauvais — APPROVED** [review]

**Review by vysakh-menon-aot — APPROVED** [review]

File: legal-api/tests/unit/services/test_authorization.py (hunk 1 above)
Comment (argush3) [reply legal-api/tests/unit/services/test_authorization.py:1517]: yes, i think so.  one sec.

Comment (argush3) [reply legal-api/tests/unit/services/test_authorization.py:1517]: removed

**Pull request merged**